workspaces/meewee-time-registration/plugins/plugin-meewee-time-registration/meewee-time-registration/src/components/ui/ui.style.ts

```diff
@@ -0,0 +1,200 @@
+import { makeStyles } from '@material-ui/core';
+import { Theme } from '@mui/material/styles';
+
+export const useCustomHeaderStyles = makeStyles(() => ({
+  '@font-face': {
+    fontFamily: 'InterFont',
```

Comment:
Fonts should be driven by the Backstage theme and should not come from the plugin itself. This create a poor UX as you have potentially very clashing designs.

Reply:
That goes for all cases of this in this file and anywhere else

Reply:
Done👍